### ISSUE TITLE
acceleration/deceleration resets to zero.

### ISSUE THREAD

**Issue by JonaldJohnston** [luni64/TeensyStep#66]
im having a weird issue where the acceleration/deceleration stages constantly reset to zero for some reason. 
here's a quick sketch of the observed motion that i am seeing:
![image](https://user-images.githubusercontent.com/38843705/80125839-d2974780-8557-11ea-8675-792c657d811d.png)
It seems to reset between 60000 and 70000 steps/s
Im using a Teensy 3.6
It happens on all pins i have tried, though some pin pairs don't work for some reason, like 2,3 or 7,8

**Comment by luni64** [comment]
Can you post a minimal program showing this behavior?

**Comment by AlfaKeNTAvR** [comment]
It looks like you have to try pulsePolarity setting of stepper motor. Test .setStepPinPolarity(HIGH);  and .setStepPinPolarity(LOW);

**Comment by JonaldJohnston** [comment]
code: 
```
#include "TeensyStep.h"

Stepper motor(9,10);       // STEP pin: 2, DIR pin: 3
StepControl controller;    // Use default settings 

void setup()
{
  motor.setMaxSpeed(70000);
  motor.setAcceleration(20000);
  
}

void loop() 
{
  motor.setTargetRel(6400*100);  // Set target position to 1000 steps from current position
  controller.move(motor);    // Do the move
  delay(2000);
}
```
i tried AlfaKeNTAvR's suggestion and it doesn't change anything.

**Comment by luni64** [comment]
This is super strange. I can reproduce the behaviour with this code: 
```c++
#include "Arduino.h"
#include "TeensyStep.h"

Stepper motor(9,10);
StepControl controller;

void printPos()
{
  Serial.printf("%d\t%d\t%d\n", millis(), motor.getPosition(), controller.getCurrentSpeed());
}

void setup()
{
  while(!Serial){}

  motor.setMaxSpeed(70000);
  motor.setAcceleration(20000);

  (new IntervalTimer())->begin(printPos, 100'000);

  motor.setTargetAbs(6400 * 100);
  controller.move(motor);
}

void loop()
{
}
```
A plot of the generated data gives this. 

![image](https://user-images.githubusercontent.com/12611497/80142151-fc2d8e80-85aa-11ea-9f31-2482778a1341.png)

I'll have a closer look at it tomorrow.

**Comment by JonaldJohnston** [comment]
hey look, my crappy graph was accurate lmao! but yeah, super strange. glad to know its not my motor/driver.

**Comment by luni64** [comment]
Should be fixed, can you try v2.0.1? 
Thanks for spotting that.
